GameData/RealismOverhaul/RO_SuggestedMods/AirplanePlus/Command.cfg

```
@@ -0,0 +1,118 @@
+//Fighter Cockpit (F-15B)
+@PART[fightercockpit|fighterinlinecockpit]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+
+	%rescaleFactor = 1.32	//rescale to F-15
+
+	@title = F-15B Cockpit (Inline)
+	@manufacturer = #roMfrMD
+	
+	@crashTolerance = 9
+	%maxTemp = 480
+	%skinMaxTemp = 900 // Mach 3
+	%skinInternalConductionMult = 0.25
+	
+	!RESOURCE,* {}
+	
+	@INTERNAL,*
+	{
+		%offset = 0.0, 0.4, 0.0
+	}
+	
+	@MODULE[ModuleCommand]
+	{
+		RESOURCE
+		{
+			name = ElectricCharge
+			rate = 2.0
+		}
+	}
+
+	MODULE
+	{
+		name = ModuleFuelTanks
+		type = SM-IV
+		basemass = 0.8
+		volume = 750
+
+		TANK
+		{
+			name = ElectricCharge
+			amount = 7200
+			maxAmount = 7200
+		}
+
+		TANK
+		{
+			name = Oxygen
+			amount = 92 // 2hrs 
+			maxAmount = 92
+		}
+	}
+}
+@PART[fightercockpit]:FOR[RealismOverhaul]
+{
+	@title = F-15B Cockpit
+	
+	%CoMOffset = 0.0, 3.5, 0.0
+	
+	@MODULE[ModuleFuelTanks]
+	{
+		@basemass = 0.9
+		@volume = 1000
+	}
+}
+
+//old fighter cockpit (P-40), Zero cockpit (A6M)
+@PART[oldfightercockpit|zerocockpit]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	
+	@title = P-40 Cockpit
+	@manufacturer = #roMfrCW
+	
+	@mass = 0.35
+	@crashTolerance = 9
+	%maxTemp = 405
+	%skinMaxTemp = 415
+	%skinInternalConductionMult = 0.25
+	
+	!RESOURCE,* {}
+	
+	@MODULE[ModuleCommand]
+	{
+		RESOURCE
+		{
+			name = ElectricCharge
+			rate = 1.2
+		}
+	}
+
+	MODULE
+	{
+		name = ModuleFuelTanks
+		type = SM-IV
```

Shouldn't this be SM-I for where these are being placed in the tech tree?
```suggestion
		type = SM-I
```

---

F-15B first flew in 1972

---

It didn't show up well as a comment. Line 35 is the F-15B, line 95 is under the P-40 and A6M config. This was for line 95.